js/alert.js

```diff
@@ -0,0 +1,18 @@
+export const showAlert = (message) => {
+  const alertContainer = document.createElement('div');
+  alertContainer.style.zIndex = 100;
```

Comment:
```suggestion
  alertContainer.style.zIndex = '100';
```

стиль он такой коварный, он только со строками работает

Reply:
поправь остальное